### ISSUE TITLE
Checkbox: storybook example labels

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#23130]
There are three example checkboxes that should have their labels updated, since currently the label text duplicates role or state information:
- "Checkbox" (there are two)
- "Required"
